shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/algorithm/sharding/datetime/AutoIntervalShardingAlgorithmTest.java

```diff
@@ -119,4 +119,40 @@ public void assertRangeDoShardingWithGreaterTenTables() {
         Collection<String> actual = shardingAlgorithm.doSharding(availableTargetNames, new RangeShardingValue<>("t_order", "create_time", Range.closed("2020-01-01 00:00:00", "2020-01-01 00:00:10")));
         assertThat(actual.size(), is(11));
     }
+
+    @Test
+    public void assertRangeDoShardingInValueWithMilliseconds() {
+        AutoIntervalShardingAlgorithm shardingAlgorithm = new AutoIntervalShardingAlgorithm();
+        shardingAlgorithm.getProps().setProperty("datetime-lower", "2020-01-01 00:00:00");
+        shardingAlgorithm.getProps().setProperty("datetime-upper", "2020-01-01 00:00:30");
+        shardingAlgorithm.getProps().setProperty("sharding-seconds", "1");
+        shardingAlgorithm.init();
+        List<String> availableTargetNames = new LinkedList<>();
+        for (int i = 0; i < 32; i++) {
```

Comment:
If needed, I can convert to `List<String> availableTargetNames = IntStream.range(0, 32).mapToObj(i -> "t_order_" + i).collect(Collectors.toCollection(LinkedList::new));`

Reply:
I think it's acceptable and doesn't need to be changed.